packages/mdc-select/_mixins.scss

```diff
@@ -163,18 +162,18 @@
 }
 
 @mixin mdc-select-ink-color_($color) {
-  .mdc-select__native-control,
```

Comment:
Why are these styles removed?

Reply:
The native control class was removed in #4797. @joyzhong to confirm?